src/Maker/MakeSerializerNormalizer.php

```diff
@@ -94,34 +88,17 @@ public function configureDependencies(DependencyBuilder $dependencies): void
         );
     }
 
-    private function generateNormalizer(string $className, Generator $generator): void
+    private function generateNormalizer(ClassNameDetails $className, Generator $generator): void
     {
         $useStatements = new UseStatementGenerator([
             NormalizerInterface::class,
-            CacheableSupportsMethodInterface::class,
+            Autowire::class,
+            sprintf('App\Entity\%s', str_replace('Normalizer', '', $className->getShortName())),
```

Comment:
What if we use `class_exists` to see if this is actually a class that exists? If it is, we do what we do now. If it is NOT, we leave the related methods entirely empty in the normalizer and allow the user to fill them in.

Reply:
good idea! && done